sdk/messaging/azeventhubs/internal/auth/token.go

```diff
@@ -0,0 +1,58 @@
+// Package auth provides an abstraction over claims-based security for Azure Event Hub and Service Bus.
+package auth
+
+//	MIT License
```

Comment:
Why does this file have a different license block format?

Reply:
Will remove. It's also in SB so I can trim it there as well.

Reply:
(I'll create a separate PR to merge up whatever we change here)

Reply:
(fixed here now)